cluster/sync.sh

```diff
@@ -1,15 +1,6 @@
 #!/bin/bash
-set -ex
 
-cluster/vagrant/sync_config.sh
-
-make all DOCKER_TAG=devel
-
-for VM in `vagrant status | grep -v "^The Libvirt domain is running." | grep running | cut -d " " -f1`; do
-  vagrant rsync $VM # if you do not use NFS
-  vagrant ssh $VM -c "cd /vagrant && export DOCKER_TAG=devel && sudo -E hack/build-docker.sh"
-done
-
-export KUBECTL="cluster/kubectl.sh --core"
-
-cluster/deploy.sh
+# This logic moved into the Makefile.
+# We're leaving this file around for people who still reference this
+# specific script in their development workflow.
+make vagrant-deploy
```

Comment:
IMHO there's few enough people involved in the project right now that we can just delete this straight away without causing any serious pain.